demos/background_subtraction_demo/python/background_subtraction_demo.py

```diff
@@ -0,0 +1,289 @@
+                         help="Optional. Merge the result frame with the original one.", action='store_true')
+    io_args.add_argument('--output_resolution', default=None, type=resolution,
+                         help='Optional. Specify the maximum output window resolution '
+                              'in (width x height) format. Example: 1280x720. '
+                              'Input frame size used by default.')
+    io_args.add_argument('-u', '--utilization_monitors', default='', type=str,
+                         help='Optional. List of monitors to show initially.')
+
+    debug_args = parser.add_argument_group('Debug options')
+    debug_args.add_argument('-r', '--raw_output_message', help='Optional. Output inference results as mask histogram.',
+                            default=False, action='store_true')
+    return parser
+
+
+def print_raw_results(outputs, frame_id):
+    scores, classes, boxes, masks = outputs
+    log.debug('  -------------------------- Frame # {} --------------------------  '.format(frame_id))
+    log.debug('  Class ID | Confidence |     XMIN |     YMIN |     XMAX |     YMAX ')
+    for box, cls, score in zip(boxes, classes, scores):
+        log.debug('{:>10} | {:>10f} | {:>8.2f} | {:>8.2f} | {:>8.2f} | {:>8.2f} '.format(cls, score, *box))
+
+
+def fit_to_window(input_img, output_resolution):
+    h, w = input_img.shape[:2]
+    target_shape = np.append(output_resolution[::-1], [3])
+    output = np.zeros(target_shape, dtype=np.uint8)
+    scale = min(output_resolution[1] / h, output_resolution[0] / w)
+    input_img = cv2.resize(input_img, None, fx=scale, fy=scale)
+    dw = output_resolution[0] - input_img.shape[1]
+    dh = output_resolution[1] - input_img.shape[0]
+    x0 = dw // 2
+    x1 = dw // 2 if dw % 2 == 0 else dw // 2 + 1
+    y0 = dh // 2
+    y1 = dh // 2 if dh % 2 == 0 else dh // 2 + 1
+    output[y0: output.shape[0] - y1, x0: output.shape[1] - x1, :] = input_img
+    return output
+
+
+def render_results(frame, objects, output_resolution, target_bgr, person_id, blur_bgr=False, show_with_original_frame=False):
+    if target_bgr is None:
+        target_bgr = np.full(frame.shape, [155, 255, 120], dtype=np.uint8)
+    else:
+        target_bgr = cv2.resize(target_bgr, (frame.shape[1], frame.shape[0]))
+    classes, masks = objects[1], objects[3]
+    # Choose masks only for person class
+    valid_inds = classes == person_id
+    masks = [mask for mask, is_valid in zip(masks, valid_inds) if is_valid]
+    if not len(masks):
+        output = frame
+    else:
+        composed_mask = masks[0]
+        for i in range(1, len(masks)):
+            composed_mask = np.logical_or(composed_mask, masks[i])
+        # Smooth contours of the predicted mask
+        composed_mask = cv2.medianBlur(composed_mask.astype(np.uint8), 11)
+        composed_mask = np.repeat(np.expand_dims(composed_mask, axis=-1), 3, axis=2)
+        if target_bgr is not None and blur_bgr:
```

Comment:
`target_bgr` can'be `None` here based on code above

Reply:
Yes, fixed.